src/lib.rs

```diff
@@ -743,16 +815,100 @@ impl From<certificate::Certificate> for Certificate {
     }
 }
 
+#[wasm_bindgen]
+pub struct StakePoolInfo(chain::stake::StakePoolInfo);
+
+impl From<chain::stake::StakePoolInfo> for StakePoolInfo {
+    fn from(info: chain::stake::StakePoolInfo) -> StakePoolInfo {
+        StakePoolInfo(info)
+    }
+}
+
+#[wasm_bindgen]
+impl StakePoolInfo {
```

Comment:
damn sorry, I wasn't fast enough to make the change here before the binding, but this will change&stabilise soon